src/apps/backend/modules/application/repository.py

```diff
@@ -27,7 +29,7 @@ def get_client(cls) -> MongoClient:
   def _create_client() -> MongoClient:
     connection_uri = ConfigService.get_string('MONGODB_URI')
     Logger.info(message=f"connecting to database - {connection_uri}")
-    client = MongoClient(connection_uri, server_api=ServerApi('1'))
+    client = MongoClient(connection_uri, server_api=ServerApi('1'), tlsCAFile=certifi.where())
```

Comment:
Why is this change needed?

Reply:
I was facing the Mongo SSL certificate issue. So that's why I added this change. For more reference please see [this](https://www.mongodb.com/community/forums/t/serverselectiontimeouterror-ssl-certificate-verify-failed-trying-to-understand-the-origin-of-the-problem/115288).

Reply:
We should understand why other products, engineers or platform has not baked this into the boilerplate. Can you please start a thread with platform team to understand this better, and either (a) Get similar change merged in boilerplate (b) Remove it from here.

Reply:
sure